pyaedt/modeler/Circuit.py

```diff
@@ -314,25 +314,25 @@ def rotate(self, selections, degrees=90):
         return True
 
 
-class ModelerSimplorer(ModelerCircuit):
-    """ModelerSimplorer class.
+class ModelerTwinBuilder(ModelerCircuit):
```

Comment:
@maxcapodi78 I am not sure if we need a deprecation warning here.

Reply:
No, since it is an internal class and cannot be called by users.